rs/ic_os/os_tools/guest_vm_runner/src/guest_vm_config.rs

```diff
@@ -103,20 +103,30 @@ pub fn generate_vm_config(
         node_type,
     );
 
-    let cpu_domain = if config.hostos_settings.vm_cpu == "qemu" {
-        "qemu"
-    } else {
-        "kvm"
+    #[cfg(feature = "dev")]
+    let (cpu_domain, vm_memory, nr_of_vcpus) = {
+        let cpu_domain = if config.hostos_settings.hostos_dev_settings.vm_cpu == "qemu" {
+            "qemu".to_string()
+        } else {
+            "kvm".to_string()
+        };
+
+        let vm_memory = config.hostos_settings.hostos_dev_settings.vm_memory;
+        let vm_nr_of_vcpus = config.hostos_settings.hostos_dev_settings.vm_nr_of_vcpus;
+
+        (cpu_domain, vm_memory, vm_nr_of_vcpus)
     };
+    #[cfg(not(feature = "dev"))]
+    let (cpu_domain, vm_memory, nr_of_vcpus) = ("kvm".to_string(), 490, 64);
```

Comment:
These values are used for prod. I've left the memory the same for now, so we can change it in a less complex commit.